index.html

```diff
@@ -388,31 +391,35 @@ <h4 class="bolda">About me</h4>
       <div class="formdiv">
         <h2 class="footer-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch<span class="touch">touch.</span></h2>
-           <form class="formx" action="https://formspree.io/f/xayagqrn" method="post">
-              <ul class="formul">
-                <li class="for for1">
-                  <label for="fullNameDesk" class="full">
-                <input type="text" name="full-name" maxlength="30" id="fullNameDesk" required>
-                  </label>
-                  <label for="lastName" class="lastx">
-                <input type="text" name="last-name" placeholder="  Last name" maxlength="30" id="lastName" required>
-                  </label>
-                </li>
-                <li class="for">
-                  <label for="emaildesk" class="eml">
-                <input type="email" name="email" id="emaildesk" placeholder="Email address" required>
-                  </label>
-                </li>
-                <li class="textare">
-                  <label for="messagedesk">
-                <textarea name="message" id="messagedesk" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
-                  </label>
-                </li>
-                <li class="buttonsub">
-                  <button type="submit" class="button_form">Get in touch</button>
-                </li>
-            </ul>
-          </form>
+        <form class="formx" id="form"  action="https://formspree.io/f/xayagqrn" method="post">
+          <ul class="formul">
+            <li class="for for1 form-control">
+              <label for="fullNameDesk" class="full form-control">
+              <input type="text" name="full-name" placeholder="   First name" maxlength="30" id="fullNameDesk" required>
+              <i class="fas fa-check-circle"></i> <i class="fas fa-times"></i>  
+              </label>
+              <label for="lastName" class="lastx form-control success">
+               <input type="text" name="last-name" placeholder="  Last name" maxlength="30" id="lastName" required>
+               <i class="fas fa-check-circle"></i><i class="fas fa-times"></i>
+              </label>
+            </li>
+            <li class="for" >
+              <label for="emaildesk" class="eml form-control error">
+                 <input type="email" required name="email" id="emaildesk" placeholder="Email address">
+                 <i class="fas fa-check-circle circle" id="circle"></i> <i class="fas fa-times time" id="time"></i>
+                </label>
+            </li>
+            <li class="textare">
+              <label for="messagedesk">
+            <textarea name="message" id="messagedesk" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+              </label>
+            </li>
+            <li><small id="errorText">The email address must contain only lowercase letters.</small></li>
```

Comment:
Your error message containts a font family that isn't requested by the guideliness. Kindly work on it:


<img width="674" alt="Screen Shot 2021-11-19 at 6 59 43" src="https://user-images.githubusercontent.com/22258274/142626849-67921df6-fe0d-4a2c-999a-570e66ea5f73.png">

- Fit the error message with the design from the chosen template in Figma (font, colors, margins).